IBAnimatableApp/IBAnimatableApp/AppDelegate.swift

```diff
@@ -38,3 +38,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   }
 
 }
+
+/// Swift < 4.2 support
+#if !(swift(>=4.2))
```

Comment:
Shall we put those in one place?

Reply:
These are for the demo app, they shouldn't influence the code of the library itself.